include/dali/core/tensor_view.h

```diff
@@ -328,6 +328,23 @@ struct TensorListViewBase {
     calculate_pointers(this->data, data, shape);
   }
 
+  bool is_contiguous() const {
+    if (num_samples() < 2)
```

Comment:
I believe this function needs unit testing. It's not a trivial function and has corner cases